bundles/k8s-rhel7/Makefile

```diff
@@ -1,15 +1,16 @@
-build:
-	docker build -t quay.io/replicated/k8s-packages:rhel-7-v1.9.3 .
-	docker build -t quay.io/replicated/k8s-packages:rhel-7-v1.9.3.20180227 .
+DATE=$(shell date +%Y%m%d)
 
-push:
-	docker push quay.io/replicated/k8s-packages:rhel-7-v1.9.3
-	docker push quay.io/replicated/k8s-packages:rhel-7-v1.9.3.20180227
+v1.9.3:
```

Comment:
It looks like we're no longer pushing tags without the date - that's intentional?

Reply:
Yes, originally we only included the OS in the tag, but it turned out the image has to be rebuilt once every couple months when a package inconsistency pops up. Adding tags seems like a sane way to avoid overwriting images that have already been QA'd and released.